messages/es.json

```diff
@@ -19,6 +18,7 @@
   "admin/savings.title": "Ahorros",
   "admin/installments.description": "Valores disponibles para interpolación: '{installmentsNumber}, {installmentValue}, {installmentsTotalValue}, {interestRate}, {hasInterest}'",
   "admin/installments.title": "Cuotas",
+  "admin/installments-list.title": "Lista de Cuotas",
```

Comment:
I don't talk spanish, so I'm not sure if this is correct

Reply:
cc @dvm1984

Reply:
@diana-cartagena